spec/udap_security_test_kit/authorization_code_redirect_test_spec.rb

```diff
@@ -0,0 +1,89 @@
+require_relative '../../lib/udap_security_test_kit/authorization_code_redirect_test'
+
+RSpec.describe UDAPSecurityTestKit::AuthorizationCodeRedirectTest, :redirect do
```

Comment:
This should be :request, not :redirect, then you won't need to include the rack test stuff.

Reply:
Got it, misunderstood that. I re-deleted the `request_helper.rb` file, updated `:redirect` to `:request`, and removed the references to `include Rack::Test::Methods`, but I'm still missing something because the unit tests are failing now with the following error:
```
Failure/Error: get "/custom/udap_security/redirect?state=#{state}"
NoMethodError:
       undefined method `get' for #<RSpec::ExampleGroups::UDAPSecurityTestKitAuthorizationCodeRedirectTest
```

What else do I need to do here? I pushed the most recent updates with failing unit tests.

Reply:
Sorry, there's a spec helper change you need, too, like this:

https://github.com/inferno-framework/subscriptions-test-kit/pull/13/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94

Reply:
Thanks. I made that change and now almost all the tests (116 out of 131) fail with the error:
```
Failure/Error: DatabaseCleaner.cleaning { example.run }
       
       StandardError:
         No suite id defined. Add `let(:suite_id) { 'your_suite_id' }` to the spec
```
Just want to confirm that I do in fact need to add this to all the rspec tests now before I do it - or is there a different underlying problem?

Reply:
Ugh, sorry, yes, that too. Basically, this new file defines context which will be included in spec that describes a runnable:

https://github.com/inferno-framework/inferno-core/blob/main/spec/runnable_context.rb

So any lets present in that file, as well as the run method, no longer need to be defined in the individual specs, but it all depends on the suite id being defined in order for everything to work.

Reply:
Alright, added `suite_id` to all the necessary spec files and all unit tests are passing - hopefully we're good to go now.